getting_started/step_by_step/signals.rst

```diff
@@ -344,6 +449,18 @@ To emit a signal in your scripts, call ``emit_signal()``.
         if health <= 0:
             emit_signal("health_depleted")
 
+ .. code-tab:: csharp C#
+
+    public void TakeDamage(int amount)
+    {
+        Health -= amount;
+
+        if (Health < 0)
+        {
+            EmitSignal("HealthDepleted");
```

Comment:
We recommend using `nameof` for the method name as it's less error-prone.

```suggestion
            EmitSignal(nameof(HealthDepleted));
```

Reply:
done